Advanced_Projects/Resume Scanner/resume_scanner.py

```
@@ -0,0 +1,43 @@
+
+nlp = spacy.load("en_core_web_sm")
+
+def extract_email(text):
+    match = re.search(r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.\w{2,}\b", text)
+    return match.group(0) if match else None
+
+def extract_phone(text):
+    match = re.search(r"(\+?\d{1,3}[\s-]?)?(\(?\d{3}\)?[\s-]?)?\d{3}[\s-]?\d{4}", text)
+    return match.group(0) if match else None
+
+def extract_name(text):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)